interpolation.py

```diff
@@ -0,0 +1,189 @@
+import numpy as np
+
+
+class VerticalGrid:
+    """
+    Can be used independently to interpolate, or used inside functions
+    in operators.py.
+
+    model_ch4_layers:  nobs x n_model_edges-1,
```

Comment:
We may need to accept a different structure if we want to use this for 72 -> 47 layer interpolations. Or, we can reshape to the GEOS-Chem grid after this step.